system/jlib/jutil.hpp

```diff
@@ -360,10 +360,18 @@ extern jlib_decl bool getConfigurationDirectory(const IPropertyTree *dirtree, //
                                                 const char *instance, 
                                                 StringBuffer &dirout);
 
-extern jlib_decl bool querySecuritySettings(bool *          _useSSL,
+extern jlib_decl bool querySecuritySettings(SSLCfg *        _useSSL,
                                             unsigned short *_port,
                                             const char * *  _certificate,
-                                            const char * *  _privateKey);
+                                            const char * *  _privateKey,
+                                            const char * *  _passPhrase);
+
+extern jlib_decl bool queryDafsServerSecSettings(SSLCfg *        _useSSL,
+                                                 unsigned short *_port,
+                                                 unsigned short *_sslport,
+                                                 const char * *  _certificate,
+                                                 const char * *  _privateKey,
+                                                 const char * *  _passPhrase);
```

Comment:
Ok, the 2 methods are partly confusing, because they are both related to dafilesrv, but only 1 sounds like it.

Maybe not a change for now, but I think it would be better if 1 method to retrieve the info, e.g. like above and a mechanism to get the preferred port (of which unless SSLBoth there would only be 1).
e.g. client code would look like:

DaFsSettings dafsSettings;
if (getDsFsSettings(dafsSettings))
  port = dafsSettings.getPort(); // returns ssl port if SSLonly or SSLBoth.